src/store/profileSlice.js

```diff
@@ -107,3 +120,20 @@ export const updateProfile = createAsyncThunk(
     return await res.json();
   }
 );
+
+export const deleteProfile = createAsyncThunk(
+  'expense/deleteExpense',
+  async () => {
+    const token = getCookie('token');
+    const res = await fetch(`${process.env.REACT_APP_BASE_URL}/profile`, {
+      method: 'DELETE',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${token}`,
+      },
+    });
+    const data = await res.json();
+
+    return data;
```

Comment:
NB: No need to store this, change this where applicable
```suggestion
    return await res.json();
```